### PR TITLE
adding phone number optout

### PR DESCRIPTION
**Pull request by tqinli**

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -202,8 +202,18 @@
...
+            <div>
+              <input type="radio" id="email_type" name="id_type" value="email" onclick="handleIdTypeClick(this)" checked>
+              <label style="border-style: none" class="raleway-medium-stratos-12px" for="email_type">Email Address</label>&nbsp;&nbsp;
+              <input type="radio" id="phone_type" name="id_type" value="phone" onclick="handleIdTypeClick(this)">
+              <label style="border-style: none" class="raleway-medium-stratos-12px" for="phone_type">Phone Number</label>
+            </div>
+
             <div class="desktopui--ry-default-C61RwL">
-              <input type="text" style="border-style: none" class="link-i16077146939-wxkpqU raleway-medium-stratos-12px border-class-1" name="email" placeholder="{{__ "Enter your email address"}}" value="{{email}}">
+              <input type="text" id="id_input" style="border-style: none" class="link-i16077146939-wxkpqU raleway-medium-stratos-12px border-class-1" name="email" placeholder="{{__ "Enter your email address"}}" value="{{email}}">
```
Comment (atarassov-ttd) [inline views/index.hbs:216]: Should the field and the placeholder be updated here?

File: public/stylesheets/index.css [hunk 2]
```diff
@@ -51,7 +51,7 @@
   height: 40px;
   justify-content: flex-start;
   margin-right: 1.0px;
-  margin-top: 30px;
+  margin-top: 20px;
```
Comment (atarassov-ttd) [inline public/stylesheets/index.css:54]: Why is this changed?

File: src/routes/index.ts [hunk 3]
```diff
@@ -39,15 +39,21 @@ const isValidEmail = (email: string) => {
...
   email: z.string(),
   recaptcha: z.string(),
 });
 
 const handleEmailPromptSubmission: RequestHandler<{}, z.infer<typeof EmailPromptRequest>, { email: string, encrypted: string, error?: string }> = async (req, res, _next) => {
   const { email, recaptcha } = EmailPromptRequest.parse(req.body);
-  if (!isValidEmail(email)) {
-    res.render('index', { email, error : i18n.__('Please enter a valid email address') });
+  if (!isValidEmail(email) && !isValidPhone(email)) {
+    res.render('index', { email, error : i18n.__('Please enter a valid email address or phone number') });
```
Comment (haitaottd) [inline src/routes/index.ts:56]: do you need to update files under locales?

File: views/index.hbs [hunk 4]
```diff
@@ -219,6 +229,14 @@
             </div>
           </form>
           <script>
+              function handleIdTypeClick(idType) {
+                if (idType.value === "email") {
+                  document.getElementById('id_input').placeholder = "Enter your email address";
```
Comment (haitaottd) [inline views/index.hbs:234]: probably need to prepopulate both and hide one to handle locale?

File: views/index.hbs [hunk 5]
```diff
@@ -202,8 +202,18 @@
...
             <input type="hidden" name="step" value="email_prompt">
             <input type="hidden" id="recaptcha" name="recaptcha" />
+
+            <br>
+
+            <div>
+              <input type="radio" id="email_type" name="id_type" value="email" onclick="handleIdTypeClick(this)" checked>
+              <label style="border-style: none" class="raleway-medium-stratos-12px" for="email_type">Email Address</label>&nbsp;&nbsp;
+              <input type="radio" id="phone_type" name="id_type" value="phone" onclick="handleIdTypeClick(this)">
+              <label style="border-style: none" class="raleway-medium-stratos-12px" for="phone_type">Phone Number</label>
```
Comment (chaseholland) [inline views/index.hbs:212]: these should be localized, right?

File: src/routes/index.ts (hunk 3 above)
Comment (chaseholland) [inline src/routes/index.ts:56]: Should we give an example of the valid formatting? It's really frustrating to type `1-555-555-5555` and have it reject because it didn't have dashes, and you didn't know that.

File: public/stylesheets/index.css (hunk 2 above)
Comment (tqinli) [reply public/stylesheets/index.css:54]: because we added a radio button above which changed the UX layout, it is from the design

File: src/routes/index.ts (hunk 3 above)
Comment (tqinli) [reply src/routes/index.ts:56]: working on it

File: views/index.hbs (hunk 5 above)
Comment (tqinli) [reply views/index.hbs:212]: fixing

File: views/index.hbs (hunk 4 above)
Comment (tqinli) [reply views/index.hbs]: there is one input, simpler
